City-of-Helsinki/palvelutarjotin-ui
# Update all the dependencies

PT-1674.
Run the `yarn upgrade` and `yarn-upgrade-all` to update all the dependencies.

Steps done:
1. run `yarn upgrade`
2. run `yarn-upgrade-all`
3. run `yarn test` to test and watch changes
4. run `yarn typecheck` to check type issues
5. run `yarn build` to test build
6. run `yarn dev` and `yarn start` to run the application and manually test it

Fix all the occurring errors. Most notably:
1. Install a new `faker`. Update the faker usage.
2. Install a missing `18next` and `react-i18next` to fix i18n init.
3. Fix the TFunctions null-returning with /src/i18n.ts to fix i18n usage
4. Fix the `stylelint` configuration and format
5. The Testing-library needs to await every userEvent. NOTE: The act-await-issues were raised!
6. Fix the RCHC-configuration (especially for the custom serach page feature in Kultus; The page swhich have more than 5 subpages renders a search form)
